org/mozilla/jss/pkcs11/PK11Signature.c

```diff
@@ -644,11 +823,17 @@ Java_org_mozilla_jss_pkcs11_SigContextProxy_releaseNativeResources
 
     /* Free the context and the proxy */
     if(proxy->type == SGN_CONTEXT) {
-        SGN_DestroyContext( (SGNContext*)proxy->ctxt, PR_TRUE /*freeit*/);
-    } else {
+        SGN_DestroyContext( (SGNContext*)proxy->ctxt, PR_TRUE /*freeit*/); } else {
```

Comment:
Why did this get collapsed?

Reply:
I do no know how that happened, but I fixed it.